lib/l10n/app_fr.arb

```diff
@@ -243,9 +237,9 @@
       }
     }
   },
-  "playButtonLabel": "LECTURE",
+  "playButtonLabel": "LIRE",
   "@playButtonLabel": {},
-  "shuffleButtonLabel": "ALÉATOIRE",
+  "shuffleButtonLabel": "ALEATOIRE",
```

Comment:
We should choose between having accents with capital letters or not. Either ALEATOIRE and TELECHARGE or ALÉATOIRE and TÉLÉCHARGÉ. Not ALEATOIRE and TÉLÉCHARGÉ like now.

Reply:
I missed that "É", I agree.